src/gradualizer_lib.erl

```diff
@@ -25,9 +25,11 @@
 %% Number of space used when prettyprinting records for nesting
 -define(PP_RECORD_NESTING_OFFSET, 2).
 
-%% merge_with for maps. Similar to merge_with for dicts.
-%% Arguably, this function should be in OTP.
 -spec merge_with(fun(), map(), map()) -> map().
+-if(?OTP_RELEASE >= 24).
+merge_with(F, M1, M2) ->
+    maps:merge_with(F, M1, M2).
+-else.
```

Comment:
We could have used `-import(maps, [merge_with/3]).` instead. Doesn't matter though.